libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -208,6 +209,12 @@ const DropdownButton: React.FC<{
         setDisplayItemCount(displayItemCount);
     }, [subtitle]);
 
+    const tooltipAlignPoints = [
```

Comment:
Please put this constant outside of the component (to prevent unnecessary re-renders), see https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/widget/widget/OptionsMenu/OptionsMenu.tsx#L22 for example.